extension/src/help/openWebHelp.ts

```diff
@@ -26,11 +27,28 @@ export class WebHelpLibrary implements IJsonLoadable {
 	functions: Dictionary<WebHelpFunction> = {};
 	ambiguousFunctions: Dictionary<WebHelpFunction[]> = {};
 	enumerators: Dictionary<string> = {};	
-	year: string;
-	
+	_fallbackYear: string;
+	get year(): string {
+		// This was converted to a getter because WebHelpLibrary only loads once on start
+		// It would of required restart after changing the launch configuration to update the year
+		return this.getYearFromSettings() ?? this._fallbackYear;
+	}
+
+	// Issue #70 requested user configuration of the help year
+	//           this implements the proposed solution of extracting it from the launch configuration
+	private getYearFromSettings(): string|null {
+		let acad = getExtensionSettingString('debug.LaunchProgram');
+		let qualifier = /(?<=AUTOCAD\s*)20\d{2}(?!ACAD.EXE)/i;
```

Comment:
Umm, just realized this probably doesn't work for Mac. Now that I have one, I just downloaded the 2022 dmg today, guess I should go do something with it... Maybe this is why Hualin was leaning towards a new setting property instead of a using the launch config?

Reply:
I asked Mac developers. The path on Mac is sth. like this:
/Applications/Autodesk/AutoCAD 2022/AutoCAD 2022.app

Reply:
Though we could check the OS in code like this:
https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/a4b83161d97835111811c0d768431ee6a03627d1/extension/src/platform.ts#L16

But I think Hualin's suggestion is to define a new setting for OpenWebHelp, so that user can manually define what ever version that is proper to use. 

And it's also possible that a user will never use launch mode - e.g. I always use the Attach mode. In this case, the launch setting could be wrong or empty.